examples/map_feature.rs

```diff
@@ -0,0 +1,47 @@
+// This file is very useful for debugging the derive macro.
+// Doctests don't work well for that purpose.
+
+fn main() {
+    #[cfg(feature = "map")]
```

Comment:
Only build-test examples with the full feature set instead. Again less noise = better.